doc/nrf/releases_and_maturity/releases/release-notes-3.0.1.rst

```diff
@@ -0,0 +1,109 @@
+.. _ncs_release_notes_3.0.1:
+
+|NCS| v3.0.1 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v3.0.1 <ncs_release_notes_300>`.
+
+##TODO
+
```

Comment:
Bluetooth®:

Channel Sounding Controller and Host are Bluetooth qualified for the nRF54L Series.

Reply:
@yuxCai are qualified, or will be qualified?

Reply:
Please move this to the Changelog section (Example: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.6.4.html#changelog)